src/components/WebexMemberRoster/WebexMemberRoster.jsx

```diff
@@ -24,11 +24,37 @@ import useMemberships from '../hooks/useMemberships';
  */
 export default function WebexMemberRoster({destinationID, destinationType}) {
   const memberships = useMemberships(destinationID, destinationType);
-  const members = memberships.map(
+
+  const getMembersNode = (data) => (data || []).map(
     (member) => <WebexMember key={member.personID} personID={member.personID} />,
   );
 
-  return <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-roster`}>{members}</div>;
+  if (destinationType !== 'meeting') {
+    return <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-roster`}>{getMembersNode(memberships)}</div>;
+  }
+
+  const groupBy = (items, key) => items.reduce(
```

Comment:
we should put this function in a separate util library or use lodash.

Reply:
@cipak Please avoid adding lodash at all costs [see @taymoork2 comment](https://github.com/webex/components/pull/162#discussion_r592740350). In fact, please discuss with the team any changes to the project's dependencies

Reply:
Yes, we saw his comment, and I agree such changes need to be discussed first. My comment was more like an invitation for comments, I'll be more clear next time. The current implementation of separating in meeting from not in meeting members uses just the standard array filter().